### DISCUSSION TITLE
Are NodeViews draggable?

### DISCUSSION THREAD

**Discussion by zcuric** [ueberdosis/tiptap#1413] [General Questions] [answered]
Like, for example [here](https://leecason.github.io/element-tiptap/). this custom image node view is draggable in tiptap@v1.  
If I replicate this in tiptap@v2 I can't achieve the same behaviour.

**Comment by hanspagel** [comment, marked as answer]
https://www.tiptap.dev/guide/node-views/vue#dragging

Does that help?
